assignments/objects.js

```diff
@@ -19,27 +19,93 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy@psu.edu",
+  "gender": "F"
+}
+
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"  
```

Comment:
```suggestion
  "gender": "M",
  speak: function() {
    return `Hello, my name is ${this.name}`
  }
```

Reply:
Try adding the function to the object. See if you can re-render Antonietta's function the same way